.xo-config.json

```diff
@@ -14,7 +14,7 @@
 		"@typescript-eslint/no-unsafe-member-access": "off",
 		"@typescript-eslint/no-unsafe-return": "off",
 		"@typescript-eslint/no-unsafe-call": "off",
-		"unicorn/no-array-callback-reference": "warn",
+		"unicorn/no-array-callback-reference": "error",
```

Comment:
This is the default, so it can be dropped

Reply:
Thanks.